tools/cdf_creator/swe_cdf_creator.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+######swe_cdf_creator.py
+######usage: <python> <swe_cdf> <>
+
+##### Creates list of ascending list from long-term SWE VIC data.
+##### Period of time is defined in configuration file. 
+##### Applies a moving window-esque method to collect 
+##### 5 days of data at a time and then sorts that data. 
+##### February 29th data is removed before being sorted.
+##### February 28th (plus surround two days) is stored at February 29th. 
+
+import datetime as dt
+import numpy as np
+import pandas as pd
+import xray
+import os
+import time
+import math
+import gc
+import argparse
+
+from tonic.io import read_config
+
+
+#read in configuration file
+parser = argparse.ArgumentParser(description='Reorder dimensions')
+parser.add_argument('config_file', metavar='config_file', type=argparse.FileType('r'), nargs=1, help='configuration file')
+args = parser.parse_args()
+config_dict = read_config(args.config_file[0].name)
+
+#read in meterological data location
+direc = config_dict['CDF']['Historic_Direc']
+file = config_dict['CDF']['Historic_File']
+start_date = config_dict['CDF']['Start_Date']
+end_date = config_dict['CDF']['End_Date']
+latlon_list = config_dict['CDF']['LatLon_List']
+output_direc = config_dict['SWE']['Out_Dir']
+
+print(start_date)
```

Comment:
you could use the logger function to do these print statements, and then you'll know where they are coming from.